.github/workflows/ci.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
       - name: Install Go
-        uses: actions/setup-go@v4
+        uses: actions/setup-go@v5
```

Comment:
@TomerHeber WDYT about using [dependabot to update those](https://docs.github.com/en/code-security/dependabot/working-with-dependabot/keeping-your-actions-up-to-date-with-dependabot)

Reply:
I'll check it out.
Thanks for the advice.